core/src/main/java/org/hibernate/ogm/datastore/document/impl/DotPatternMapHelpers.java

```diff
@@ -130,4 +132,15 @@ public static boolean organizeAssociationMapByRowKey(
 	private static MapStorageType getMapStorage(AssociationContext associationContext) {
 		return associationContext.getAssociationTypeContext().getOptionsContext().getUnique( MapStorageOption.class );
 	}
+
+	public static String getColumnSharedPrefixOfAssociatedEntityLink(AssociationKey associationKey) {
+		String[] associationKeyColumns = associationKey.getMetadata()
+				.getAssociatedEntityKeyMetadata()
+				.getAssociationKeyColumns();
+		// we used to check that columns are the same (in an ordered fashion)
```

Comment:
that's a c'n'p leftover. I felt more safe to retail the line since I don't get it either.